pages/components/image/image.html

```diff
@@ -0,0 +1,281 @@
+                transform: scale(1);
+                overflow: hidden;
+              "
+              sandbox="allow-scripts allow-same-origin"
+            >
+            </iframe>
+          </div>
+        </div>
+        <div class="responsive-image-wrapper">
+          <h2 class="mt-4 mb-4">Responsive Images with Size Variant</h2>
+          <div class="image-wrapper p-5">
+            <img
+              class="responsive-image xl-image m-2"
+              src="https://i.pravatar.cc/300"
+              alt="responsive image"
+            />
+            <img
+              class="responsive-image lg-image m-2"
+              src="https://i.pravatar.cc/300"
+              alt="responsive image"
+            />
+            <img
+              class="responsive-image sm-image m-2"
+              src="https://i.pravatar.cc/300"
+              alt="responsive image"
+            />
+          </div>
+          <p class="mt-4 mb-4">
+            Responsive Images are the images that work well on devices with
+            widely differing screen sizes. To make your image responsive add
+            <b>'.responsive-image'</b> class to your image element and for size
+            variant you can use <b>'.xl-image'</b>, <b>'.lg-image'</b> or
+            <b>'.sm-image'</b> and you can also use paddings and margins utility
+            classes for adding spacing.
+          </p>
+          <div class="code-wrapper">
+            <iframe
+              src="https://carbon.now.sh/embed?bg=rgba%28195%2C216%2C236%2C1%29&t=panda-syntax&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cimg%2520class%253D%2522responsive-image%2520xl-image%2520m-2%2522%2520src%253D%2522https%253A%252F%252Fi.pravatar.cc%252F300%2522%2520alt%253D%2522responsive%2520image%2522%2520%252F%253E%250A%250A%253Cimg%2520class%253D%2522responsive-image%2520lg-image%2520m-2%2522%2520src%253D%2522https%253A%252F%252Fi.pravatar.cc%252F300%2522%2520alt%253D%2522responsive%2520image%2522%2520%252F%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%250A%253Cimg%2520class%253D%2522responsive-image%2520sm-image%2520m-2%2522%2520src%253D%2522https%253A%252F%252Fi.pravatar.cc%252F300%2522%2520alt%253D%2522responsive%2520image%2522%2520%252F%253E"
+              style="
+                width: 864px;
+                height: 150px;
+                border: 0;
+                transform: scale(1);
+                overflow: hidden;
+              "
+              sandbox="allow-scripts allow-same-origin"
+            >
+            </iframe>
+          </div>
+        </div>
+        <div class="round-image-wrapper">
+          <h2 class="mt-4 mb-4">Round Images with size Variant</h2>
+          <div class="image-wrapper p-5">
+            <img
+              class="responsive-image round-image xl-image m-2"
+              src="https://i.pravatar.cc/300"
```

Comment:
Does 300 refer to the size of the image here? How do your CSS classes behave when it's a larger image? Say 600 or 800x600?

Reply:
It will behave the same for any size of the image, if you want to check you can test that.